pykafka/rdkafka/simple_consumer.py

```diff
@@ -0,0 +1,152 @@
+from pykafka.simpleconsumer import (
+    SimpleConsumer, ConsumerStoppedException, OffsetType)
+from . import _rd_kafka
+
+
+class RdKafkaSimpleConsumer(SimpleConsumer):
+    """A pykafka.SimpleConsumer with librdkafka-based fetchers
+
+    This aims to conform to the SimpleConsumer interface as closely as
+    possible.  There are some notable differences:
+
+    1. rotating over partitions: while message ordering within partitions is
+       conserved (of course!), the order in which partitions are visited will
+       deviate.  In particular, here we may emit more than one message from
+       the same partition before visiting another
+    2. ignores num_consumer_fetchers: librdkafka will typically spawn at least
+       as many threads as there are kafka cluster nodes
+
+    For an overview of how configuration keys are mapped to librdkafka's, see
+    _mk_rdkafka_config_lists.
+    """
+
+    def _setup_fetch_workers(self):
+        brokers = ','.join(map(lambda b: ':'.join((b.host, str(b.port))),
+                               self._cluster.brokers.values()))
+        partition_ids = list(self._partitions_by_id.keys())
+        start_offsets = [
+            self._partitions_by_id[p].next_offset for p in partition_ids]
+        conf, topic_conf = self._mk_rdkafka_config_lists()
+
+        rdk_consumer = _rd_kafka.Consumer()
+        rdk_consumer.configure(conf=conf)
+        rdk_consumer.configure(topic_conf=topic_conf)
+        rdk_consumer.start(brokers,
+                           self._topic.name,
+                           partition_ids,
+                           start_offsets)
+        return rdk_consumer
+
+    def consume(self, block=True):
+        timeout_ms = self._consumer_timeout_ms if block else 1
+        try:
+            msg = self._fetch_workers.consume(timeout_ms)
```

Comment:
I understand why this works, but the `fetch_workers` terminology here makes this logic more confusing than it should be. I believe the naming in `SimpleConsumer` that gets overridden here should change slightly to accommodate it, if possible.